topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md

```diff
@@ -0,0 +1,172 @@
+>
+{: .hands_on}
+
+## Blast output
+
+> {% icon tool %} The BLAST output will be in tabular format (you can select the desired output format from the drop down menu) and include the following fields : 
+
+>
+
+<table>
+  <tr>
+    <th>Column</th>
+    <th>NCBI name</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>1</td>
+    <td>qseqid</td>
+    <td>Query Seq-id (ID of your sequence)</td>
+  </tr>
+  <tr>
+    <td>2</td>
+    <td>sseqid</td>
+    <td>Subject Seq-id (ID of the database hit)</td>
+  </tr>
+   <tr>
+    <td>3</td>
+    <td>pident</td>
+    <td>Percentage of identical matches</td>
+  </tr>
+   <tr>
+    <td>4</td>
+    <td>length</td>
+    <td>Alignment length</td>
+  </tr>
+   <tr>
+    <td>5</td>
+    <td>mismatch</td>
+    <td>Number of mismatches</td>
+  </tr>
+   <tr>
+    <td>6</td>
+    <td>gapopen</td>
+    <td>Number of gap openings</td>
+  </tr>
+   <tr>
+    <td>7</td>
+    <td>qstart</td>
+    <td>Start of alignment in query</td>
+  </tr>
+   <tr>
+    <td>8</td>
+    <td>qend</td>
+    <td>End of alignment in query</td>
+  </tr>
+   <tr>
+    <td>9</td>
+    <td>sstart</td>
+    <td>Start of alignment in subject (database hit)</td>
+  </tr>
+   <tr>
+    <td>10</td>
+    <td>send</td>
+    <td>End of alignment in subject (database hit)</td>
+  </tr>
+    <tr>
+    <td>11</td>
+    <td>evalue</td>
+    <td>Expectation value (E-value)</td>
+  </tr>
+    <tr>
+    <td>12</td>
+    <td>bitscore</td>
+    <td>Bit score</td>
+  </tr>
+</table>
+
+> The fields are separated by tabs, and each row represents a single hit. For more details for BLAST analysis and output, we recommand you to follow the [Similarity-searches-blast](https://training.galaxyproject.org/training-material/topics/genome-annotation/tutorials/genome-annotation/tutorial.html#similarity-searches-blast) tutorial.
+
+
+> <details-title>Further Reading about BLAST Tools in Galaxy</details-title>
+>
+>
+> Cock et al. (2015): [NCBI BLAST+ integrated into Galaxy](http://biorxiv.org/content/early/2015/05/04/014043.full-text.pdf+html)
+>
+> Cock et al. (2013): [Galaxy tools and workflows for sequence analysis with applications in molecular plant pathology](https://peerj.com/articles/167/)
+{: .details}
+
+## More Similarity Search Tools on Galaxy
+
+* **Diamond**: {% tool [Diamond](toolshed.g2.bx.psu.edu/repos/bgruening/diamond/bg_diamond/2.0.15+galaxy0) %} is a high-throughput program for alignment of  large-scale data sets. It aligns sequences to the reference database using a compressed version of the reference sequences called a "database diamond" which is faster to read and can save computational time (~20,000 times the speed of Blastx, with high sensitivity).
+
+> <details-title>Diamond in depth</details-title>
+>
+> [Buchfink et al. (2015): Fast and sensitive protein alignment using Diamond.](https://www.nature.com/nmeth/journal/v12/n1/abs/nmeth.3176.html)
+{: .details}
```

Comment:
```suggestion
> <details-title>Diamond in depth</details-title>
> See {% cite Buchfink2014 %} for more discussion.
{: .details}
```

here you can use a proper `citation`, just add the following to the `tutorial.bib` file in the same folder as your tutorial.md

```
@article{Buchfink2014,
  doi = {10.1038/nmeth.3176},
  url = {https://doi.org/10.1038/nmeth.3176},
  year = {2014},
  month = nov,
  publisher = {Springer Science and Business Media {LLC}},
  volume = {12},
  number = {1},
  pages = {59--60},
  author = {Benjamin Buchfink and Chao Xie and Daniel H Huson},
  title = {Fast and sensitive protein alignment using {DIAMOND}},
  journal = {Nature Methods}
}
```

Reply:
Thank you for your suggestion, changes have been made accordingly.

Reply:
Hi @hexylena, 
Thank you for your input. I have made the necessary changes. Can you please take a look and confirm if they are ready to be merged.